### ISSUE TITLE
Aria attributes must conform to valid values

### ISSUE THREAD

**Issue by wenche** [equinor/energyvision#1024]
Audit reports several issues here, and I'm not able to see by first glance why this is the case. But the pages listed seem to have forms in common.

![Skjermbilde 2022-04-22 kl. 11.37.24.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/f9fab7cc-f69b-49ad-996a-4c1f5feb8741)

### Acceptance criteria
- If this is an issue, it should be fixed
